CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.1.0] - 2023-11-06
+### Added
+- Added new poller thread pool usage metrics #1275
```

Comment:
`concurrent-task-running` metric is not reporting correct numbers. I told @timl3136 about it. let's fix the metrics before we release 1.1.0.

Reply:
included the rename #1291. Thank you for catching that